pkg/api/openshiftcluster.go

```diff
@@ -175,9 +175,10 @@ const (
 type MaintenanceTask string
 
 const (
-	MaintenanceTaskEverything MaintenanceTask = "Everything"
-	MaintenanceTaskOperator   MaintenanceTask = "OperatorUpdate"
-	MaintenanceTaskRenewCerts MaintenanceTask = "CertificatesRenewal"
+	MaintenanceTaskEverything   MaintenanceTask = "Everything"
+	MaintenanceTaskOperator     MaintenanceTask = "OperatorUpdate"
+	MaintenanceTaskRenewCerts   MaintenanceTask = "CertificatesRenewal"
+	MaintenanceTaskEtcdRecovery MaintenanceTask = "EtcdRecovery"
```

Comment:
We'll want this removed as we're no longer doing etcd recovery as a maintenance task.